src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -301,3 +301,42 @@ export const onRequest = defineMiddleware(async (context, next) => {
 </p>
 
 `deserializeActionResult()` will reverse the effect of `serializeActionResult()` and return an action result to its original state. This is useful to access the `data` and `error` objects on a serialized action result.
+
+### `getActionPath()`
+
+<p>
+
+**Type:** `(action: ActionClient<any, any, any>) => string`
+<Since v="5.1.0" />
+</p>
+
+The `getActionPath()` utility accepts an action and returns a URL path so you can execute an action call as a `fetch()` operation directly. This allows you to provide details such as custom headers when you call your action.
+
+This example shows how to call an action with the `Authorization` header and the [`keepalive`](https://developer.mozilla.org/en-US/docs/Web/API/Request/keepalive) option:
+
+```ts {7,9}
```

Comment:
Can we have a file title here? And, what do you think should be highlighted in the code sample? maybe just  `getActionPath(actions.like)` or would you do more?

Reply:
I'm not sure a title is relevant here because there's no context in this example. Should we add more like with a button event listener etc?

About highlighting, I'm unsure what we should emphasize. The use of  `getActionPath()`? The custom things it allows to provide?

Reply:
Well, let's start with which file would you literally write this code in?

Reply:
mmh I think it's more likely to happen server side, so for example in a .astro component script (not frontmatter)

Reply:
OK, then let's give these file names of `src/components/my-component.astro` and use the code fences at top and bottom (and just not show anything below the code fence). What about that?